tensorflow/lite/micro/kernels/BUILD

```diff
@@ -304,6 +304,7 @@ cc_library(
         "squeeze.cc",
         "strided_slice.cc",
         "sub.cc",
+		"sub_common.cc",
```

Comment:
replace tabs with spaces.

The ubuntu sanity check is failing because of this. You can also install buildifier to automatically handle this:
https://github.com/tensorflow/tensorflow/blob/1cdd4da14282210cc759e468d9781741ac7d01bf/tensorflow/tools/ci_build/install/install_buildifier.sh#L18-L21

```
buildifier tensorflow/lite/micro/kernels/BUILD
```